htdocs/PHP/internal_libraries/languages/page_content_fr.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/**
+* Translations for page contents
+* Language: French
+*/
+
+$translation = array();
+
+// Title
+$translation["title"] = "WeExpire - Plus simple qu'un testament, plus sûr qu'une note.";
+
+// Description for search engines (max 160 characters)
+// The following string is exactly 160 chars
+$translation["meta_description"] = "WeExpire est un outil permettant de créer des notes d'urgence qui ne pourront être lues par vos contacts de confiance qu'après votre décès ou en cas de blessure.";
+
+// Description for social media (max 100 characters)
+// The following string is exactly 98 chars
+$translation["social_media_description"] = "Créez des notes d'urgence qui ne pourront être lues par vos contacts proches qu'après votre décès.";
+
+// Footer section
+$translation["footer_language"] = "Langue";
+$translation["footer_about"] = "À propos";
+$translation["footer_donate"] = "Dons";
+$translation["footer_contact"] = "Contact";
+$translation["footer_legal"] = "Légal";
+$translation["footer_privacy"] = "Vie privée";
+$translation["footer_terms"] = "Termes et conditions";
+
+// Buttons
+$translation["button_get_started"] = "Commencer";
+$translation["button_create_another_one"] = "Créer une aute note";
+$translation["button_create_your_emergency_note"] = "Créer une note d'urgence";
+$translation["button_next"] = "Continuer";
+$translation["button_confirm"] = "Confirmer";
+$translation["button_get_emergency_note"] = "Obtenir une note d'urgence";
```

Comment:
This text is displayed over the button that the user will click in order to download his/her emergency note. I'm not sure but it feels to me that "une" sounds generic (at least in Italian) - would it be possible to say in French something like "Obtenir le note d'urgence"?

Reply:
Yeah you're totally right on this, I'll fix that